src/prql.pest

```diff
@@ -21,6 +21,8 @@ transformation = { WHITESPACE* ~ ident ~ expr }
 
 function = { "func" ~ idents ~ "=" ~ items }
 
+table = { "table" ~ ident ~ "=" ~ "(" ~ NEWLINE* ~ pipeline ~ NEWLINE* ~ ")" }
```

Comment:
Would this work? (the current version is fine otherwise)
```suggestion
table = { "table" ~ ident ~ "=" ~  inline_pipeline }
```

Reply:
If you use `inline_pipline`, it will stop working when a new line is inserted.
So, I decided to write it this way. (There may be a better way to write it, though.)

Reply:
Ah OK. I am working on parentheses anyway, so let's keep this like you have it and we can iterate on it later. Thanks